deployment/evm/upgrade.ts

```diff
@@ -76,11 +77,15 @@ async function upgradeProxyWithNewImplementation(
   const signer = await getSigner(chain);
   const { Tbrv3 } = await import("@xlabs-xyz/evm-arbitrary-token-transfers");
 
+  // HACK! resolveWrappedToken does not seem to work for CELO native currency.
+  const gasTokenAddress = chain.chainId === 14 ? new EvmAddress(getDependencyAddress("initGasToken", chain)) : undefined;
```

Comment:
We should check whether this works for mainnet, right? If it is defined for mainnet, I prefer grabbing this from the SDK.

Reply:
Ah, I see, then we're in the same situation.